src/babel-plugin-transform-sx-tailwind/src/__tests__/fixtures/template-literal-expression/code.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import React, { type Node } from 'react';
+import { tailwind } from '@adeira/sx-tailwind';
+import * as sx from '@adeira/sx';
+
+export default function Example(): Node {
+  return (
+    <div className={`${tailwind('px-4')} ${styles('customStyle')}`}>
+      Lorem lipsum
+    </div>
+  );
+}
+
+const styles = sx.create({
+  customStyle: {transform: 'translateX(10px)'}
+});
```

Comment:
Missing newline. I will check our Eslint config to see why this rule is missing. 🤔

Reply:
Because of `.eslintignore`. But maybe we should ignore only output files.

Reply:
@michalsanger I see! That could be it. I am OK with disabling Eslint for fixtures but would it be possible to do it locally? I mean adding `/* eslint-disable */` into each input (and therefore even output) file.

Reply:
I'll try to ignore only output files. It's useful to have input files in shape.

Reply:
As expected - couple of errors fixed in input files :-)